### ISSUE TITLE
Search don't work for one app

### ISSUE THREAD

**Issue by Henrietta1989** [mas-cli/mas#176]
Mas 1.4.1 don't find EventScripts at all. Other apps it founds normally.

mas search EventScripts
Error: No results found

https://itunes.apple.com/app/eventscripts/id525319418?mt=12

**Comment by Henrietta1989** [comment]
Mojave with mas-cli 1.4.1:

> mas search eventscipts
No results found
Error: No results found

> mas search graphicconverter
408364640 GraphicConverter (8.8.2)
736099092 GraphicConverter 9 (9.7.5)
1090470302 GraphicConverter 10 (10.6.5)

> mas reset

> mas search eventscipts
No results found
Error: No results found

> mas search 525319418
No results found
Error: No results found

> mas info 525319418
No results found
Error: No results found

**Comment by phatblat** [comment]
I get the same results using mas 1.4.3, with the exception that `mas search graphicconverter` returns nothing (yet the command reports success, see #182).

However, when I click the iTunes link in the description I get an error in the MAS GUI:

![screen shot 2018-10-25 at 7 55 58 pm](https://user-images.githubusercontent.com/28851/47541737-a84bc780-d897-11e8-8112-4fe8b7616e00.png)

> I'm signed into the MAS GUI and it works normally otherwise.

Searching for "eventscripts" in the MAS GUI returns no results:

<img width="1112" alt="screen shot 2018-10-25 at 8 49 14 pm" src="https://user-images.githubusercontent.com/28851/47541742-aeda3f00-d897-11e8-87d7-4b47fa126932.png">

Is EventScripts an app you created? Is it published? What country/region are you using? I'm testing this in the United States and the app might not be published to that country/region.

**Comment by Henrietta1989** [comment]
I'ts not my app and it is published see urls below. I'm not sure what region it is but i think developer is in London.

[http://www.mousedown.net/mouseware/EventScripts.html](url)
[https://itunes.apple.com/app/eventscripts/id525319418?mt=12](url)
info@mousedown.net

**Comment by vimusr** [comment]
It could be that this app actually isn't available via the MAS anymore, as their webpage was last updated in 2014 and many of their apps refer to mac OS X 10.6~10.8. Maybe the app is dated and no longer available via the App Store?

**Comment by vimusr** [comment]
I contacted the developer of EventScripts in Nov. of 2018--he just got back to me and confirmed he updated the app and it is now included in the US MAS gui as of April 2019.
<img width="707" alt="Screen Shot 2019-04-09 at 11 00 22 AM" src="https://user-images.githubusercontent.com/23594932/55823673-19361300-5ab7-11e9-87c6-d242896548a4.png">

mas search appears to still find no results, however. 

Just thought I'd pass along this update.

**Comment by vhemt** [comment]
Same problem for me with Reeder 4:
```
mas search magnet
   441258766  Magnet                                                  (2.4.3)
   716628309  An Alien with a Magnet HD                               (1.0.0)
  1032456933  Magnetola - Vintage Cassette Player with Sound Softener (1.0.8)
   446490677  Edit In The Magnetic Timeline                           (7.1)
```

```

**Comment by phatblat** [comment]
Fixed in #269
